src/cmd/run.go

```diff
@@ -475,6 +483,10 @@ func constructExecArgs(container string,
 
 	execArgs = append(execArgs, envOptions...)
 
+	if runFlags.preserve_fds != 0 {
+		execArgs = append(execArgs, "--preserve-fds="+strconv.Itoa(runFlags.preserve_fds))
```

Comment:
This part will require rebasing against `main`.

Reply:
git seems to have done a good enough job here, on its own.

Reply:
Yes, indeed.  :)